src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import navbar from './navbar';
+import banner from './banner';
+import cuisinelist from './cuisinelist';
+import cuisine from './cuisine';
+import chef from './chef';
+import menu from './menu';
+import contact from './contact';
+
+const bannerEl = banner();
+const menuEl = menu();
+const contactEl = contact();
+
+document.body.prepend(navbar());
+const homeBtn = document.getElementById('home');
+const menuBtn = document.getElementById('menu');
+const contactBtn = document.getElementById('contact');
+
+const parent = document.createElement('div');
+parent.appendChild(bannerEl);
+parent.appendChild(menuEl);
+parent.appendChild(contactEl);
+
+const varcuisine = cuisinelist();
+document.getElementById('content').appendChild(parent);
+document.getElementById('content').appendChild(cuisine(varcuisine));
+document.getElementById('content').appendChild(chef());
+
+homeBtn.onclick = () => {
+  bannerEl.classList.add('banner-show');
+  menuEl.classList.remove('banner-show');
+  contactEl.classList.remove('banner-show');
+  homeBtn.classList.add('active');
+  menuBtn.classList.remove('active');
+  contactBtn.classList.remove('active');
+};
+
+menuBtn.onclick = () => {
+  bannerEl.classList.remove('banner-show');
+  menuEl.classList.add('banner-show');
+  contactEl.classList.remove('banner-show');
+  homeBtn.classList.remove('active');
+  menuBtn.classList.add('active');
+  contactBtn.classList.remove('active');
+};
+
+contactBtn.onclick = () => {
+  bannerEl.classList.remove('banner-show');
+  menuEl.classList.remove('banner-show');
+  contactEl.classList.add('banner-show');
+  homeBtn.classList.remove('active');
+  menuBtn.classList.remove('active');
+  contactBtn.classList.add('active');
```

Comment:
- [ ] You have a lot of repetition of codes here. In order to write clean and organized codes and also to maintain the DRY rule, you should ensure a piece of code that is reused in several parts of your project is encapsulated in a function of its own. In this way, you do not have to repeat yourself every time. 
The highlighted piece of code can be refactored into its own function in several ways. One of them can be like this - 

```
const displayOnClicked = (ind) => {
  const pageEl = [bannerEl, menuEl, contactEl];
  const buttons = [homeBtn, menuBtn, contactBtn];
  pageEl.forEach((li, index) => {
    if (index === ind) {
      li.classList.add('banner-show');
      buttons[index].classList.add('active');
    } else {
      li.classList.remove('banner-show');
      buttons[index].classList.remove('active');
    }
  })
}
```
From the above method, you defined a parameter for your function which takes in the index of element or button which should be added a classist or removed a classist. Then, you defined the pageElements and Buttons in a list, loop through them and if the index of any of the looped elements is the same as the parameter defined, you add the required classList and if not, you remove it. 
With this, you can call the function in the appropriate clicked events like so - 

```
homeBtn.onclick = () => {
  displayOnClicked(0);
};

menuBtn.onclick = () => {
  displayOnClicked(1);
};
```

Reply:
Thanks a lot for the suggestions, I have made the changes you asked here for index.js.